diff --git a/cpclib-runner/src/runner/tracker/at3.rs b/cpclib-runner/src/runner/tracker/at3.rs
index 2bb63a5d..a1fc036a 100644
--- a/cpclib-runner/src/runner/tracker/at3.rs
+++ b/cpclib-runner/src/runner/tracker/at3.rs
@@ -14,6 +14,7 @@ pub const AT_CMD: &str = "at3";
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Default)]
 pub enum At3Version {
     #[default]
+    V3_5_1A,
     V3_5,
     V3_4,
     V3_3,
@@ -24,6 +25,7 @@ pub enum At3Version {
 impl Display for At3Version {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let v = match self {
+            At3Version::V3_5_1A => "v3.5.1a",
             At3Version::V3_5 => "v3.5",
             At3Version::V3_4 => "v3.4",
             At3Version::V3_3 => "v3.3",
@@ -38,6 +40,16 @@ impl Display for At3Version {
 impl StaticInformation for At3Version {
     fn static_download_urls(&self) -> &'static crate::delegated::MutiplatformUrls {
         match self {
+            At3Version::V3_5_1A => {
+                static URL: OnceLock<MutiplatformUrls> = OnceLock::new();
+                URL.get_or_init(|| {
+                MutiplatformUrls::builder()
+                    .linux("https://www.julien-nevo.com/arkostracker/release/3.5.1a/linux64/ArkosTracker-linux64-3.5.1a.zip")
+                    .windows("https://www.julien-nevo.com/arkostracker/release/3.5.1a/windows/ArkosTracker-windows-3.5.1a.zip")
+                    .macos("https://www.julien-nevo.com/arkostracker/release/3.5.1a/windows/ArkosTracker-windows-3.5.1a.zip")
+                    .build()
+                })
+            },
             At3Version::V3_5 => {
                 static URL: OnceLock<MutiplatformUrls> = OnceLock::new();
                 URL.get_or_init(|| {
